### DISCUSSION TITLE
Z-shell should accept finger print when using a sudo command

### DISCUSSION THREAD

**Discussion by Touseef-ahmad** [ohmyzsh/ohmyzsh#10254] [Ideas]
I am using MacBook Pro 2017, that has finger print option, it is much easier than entering a password every time you are doing something that requires admin permissions. If that could be added in z-shell that would make life so much easier. Whenever I have to use a sudo command, I would use my finger instead of password.

**Comment by mcornella** [comment]
That is a `sudo` command feature request, not Zsh. I haven't look at this a lot but it looks like it might be possible: https://askubuntu.com/questions/1015416/use-fingerprint-authentication-not-only-for-login.
